packages/safe-core-sdk/src/safeFactory/index.ts

```diff
@@ -140,13 +147,45 @@ class SafeFactory {
     ])
   }
 
+  async predictSafeAddress({
+    safeAccountConfig,
+    safeDeploymentConfig
+  }: PredictSafeProps): Promise<string> {
+    validateSafeAccountConfig(safeAccountConfig)
+    if (safeDeploymentConfig) {
+      validateSafeDeploymentConfig(safeDeploymentConfig)
+    }
+
+    const from = this.#safeProxyFactoryContract.getAddress()
+
+    const initializer = await this.encodeSetupCallData(safeAccountConfig)
+    const saltNonce = safeDeploymentConfig.saltNonce.toString()
```

Comment:
can you make it a string in the config? it's a `uint256` and javascript number cannot fit it